src/components/molecules/PortalItem/PortalItem.tsx

```diff
@@ -92,7 +98,16 @@ export const PortalItem: React.FC<PortalItemProps> = ({
 
     if (template !== "external") {
       const venueData = buildEmptyVenue(venueName, template);
-      await createVenue_v2({ ...venueData, worldId, parentId: venueId }, user);
+      await createVenue_v2(
+        {
+          ...venueData,
+          worldId,
+          parentId: venueId,
+          showShoutouts: DEFAULT_SHOW_SHOUTOUTS,
+          showReactions: DEFAULT_SHOW_REACTIONS,
```

Comment:
Why isn't this dealt with in `createVenue_v2()` or in the clould function in case `showShoutouts` and `showReactions` are missing or undefined?

In short: why is it the front-end component concern to provide the defaults?

Reply:
yeah, i agree
moved them to `createVenue_v2` in 17a8f28

Reply:
I guess it is enough of an improvement.

It still doesn't solve any possible BE issues and it may be a gray area about should we always save some values or not.